### PR TITLE
security&monitor&resource&parts of compute

### PR DESCRIPTION
**Pull request by rosasu**

### CODE REVIEW

File: content/compute/hpc/intro/advantage.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ weight: 20
 
 ## 弹性基础设施，提高工作效率
 
-QingCloud HPC消除了通常与本地 HPC 集群相关的漫长等待时间和生产力损失。另外 QingCloud HPC还会提供集成服务套件，包含快速、轻松地在云中构建和管理 HPC 集群所需的一切，让各种垂直行业能够将其用于运行最为计算密集型的工作负载。让用户可以很大程度避免自建超算系统带来的诸多不便，极大的提高工作效率。
+shanhe HPC消除了通常与本地 HPC 集群相关的漫长等待时间和生产力损失。另外 shanhe HPC还会提供集成服务套件，包含快速、轻松地在云中构建和管理 HPC 集群所需的一切，让各种垂直行业能够将其用于运行最为计算密集型的工作负载。让用户可以很大程度避免自建超算系统带来的诸多不便，极大的提高工作效率。
```
Comment (pethu1110) [inline content/compute/hpc/intro/advantage.md:12]: shanhe？

File: content/ai/deep_learning/intro/intro.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: "什么是深度学习平台"
+description: 
+draft: false
+enableToc: false
+weight: 10
+keyword: 
+---
+
+本文档仅对应青云深度学习平台 2.0 版本。
```
Comment (pethu1110) [inline content/ai/deep_learning/intro/intro.md:10]: 青云？

File: content/ai/deep_learning/intro/intro.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Keras 是一个高层神经网络 API，由 Python 编写，通过调用其他深度学习框架来进行计算，如 TensorFlow、 Theano 以及 CNTK。
+
+ Keras 高度模块化，能快速搭建神经网络，并且非常容易上手。
+
+## 青云深度学习平台
+
+深度学习平台基于 Docker 容器技术，将主流的深度学习框架做成 docker 镜像的形式，用户无需为环境进行繁琐的配置，直接运行容器化的深度学习应用，即可迅速开展训练和预测任务，提高了用户的开发和部署效率。
+
+青云深度学习平台提供入门版、基础版和企业版三个版本。其中，
```
Comment (pethu1110) [inline content/ai/deep_learning/intro/intro.md:46]: 青云？

File: content/ai/deep_learning/manual/TensorFlow.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+### 分布式
+
+增加节点，在线扩容。在详情页点击**新增节点**按钮，可以对每个新增节点指定 IP 或选择自动分配。
+
+TensorFlow 分布式训练需要指定 parameter server 和 worker 的 IP 地址和端口号（根据自己的 IP 进行修改）
+
+#### 跨区分布式
+
+青云深度学习平台支持跨区分布式TensorFlow训练，首先使用IPSec或GRE方式，连通两个集群的路由器。参考[IPSec隧道](https://docs.qingcloud.com/product/network/ipsec)。如果是异地路由器，则要求两个路由器都有公网IP地址，并为公网IP分配足够的带宽，依据实际训练数据交互的带宽需求，调整带宽到合适的值。两个路由器连通之后，集群中的深度学习节点将会在不同的网段，例如 192.168.1.2 和 192.168.2.2 但是相互之间的连通性和在一个局域网没有差别，在进行 TensorFlow 分布式训练时，只需按照实际地址，指定分布式训练节点 IP 地址即可。
```
Comment (pethu1110) [inline content/ai/deep_learning/manual/TensorFlow.md:72]: 青云？

File: content/ai/deep_learning/manual/pyTorch.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+<img src="../../_images/pytorchdist_start.png" style="zoom:60%;" />
+
+#### PyTorch 分布式训练结果
+
+<img src="../../_images/pytorchdist_result.png" style="zoom:60%;" />
+
+#### 跨区分布式
+
+青云深度学习平台支持跨区分布式 PyTorch 训练，首先使用 IPSec 或 GRE 方式，连通两个集群的路由器。参考[IPSec隧道](https://docs.qingcloud.com/product/network/ipsec)。如果是异地路由器，则要求两个路由器都有公网 IP 地址，并为公网 IP 分配足够的带宽，依据实际训练数据交互的带宽需求，调整带宽到合适的值。两个路由器连通之后，集群中的深度学习节点将会在不同的网段，例如 192.168.1.2 和 192.168.2.2，但是相互之间的连通性和在一个局域网没有差别。
```
Comment (pethu1110) [inline content/ai/deep_learning/manual/pyTorch.md:64]: 青云

File: content/ai/deep_learning/manual/testing.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+title: "平台性能测试指南"
+description: test
+weight: 80
+draft: false
+---
+
+为了让用户快速了解深度学习平台的性能指标以便于选择，我们使用 TensorFlow [性能测试代码](https://github.com/tensorflow/benchmarks)中的 CNN Benchmark ，对常用硬件配置组合进行了测试。
+我们选择 ResNet-50 模型，调用 Python 3.6 + TensorFlow 1.7.0（ ROCm 版为 1.14.0 ） 环境，使用合成数据进行测试。
+
+**青云深度学习平台性能测试表**
```
Comment (pethu1110) [inline content/ai/deep_learning/manual/testing.md:12]: 青云？

File: content/compute/hpc/intro/advantage.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ EHPC相较传统HPC，具有灵活的配置和几乎无限的可扩展性使用
 
 ## 界面化软件管理
 
-在QingCloud HPC，您可以一键完成软件安装、卸载、升级等操作，并完成环境变量的设定，不需要长时间等待软件的安装，无需担心安装失败问题。
+在shanhe HPC，您可以一键完成软件安装、卸载、升级等操作，并完成环境变量的设定，不需要长时间等待软件的安装，无需担心安装失败问题。
```
Comment (pethu1110) [inline content/compute/hpc/intro/advantage.md:25]: shanhe?

File: content/compute/hpc/intro/intro.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ weight: 10
 ---
 
 
-HPC 高性能计算是青云QingCloud推出的一站式超算平台，提供弹性和可扩展的HPC计算资源，是传统HPC技术与云计算技术的融合升级。提供丰富多样的使用模式适配多种应用场景，统一的管理系统提供最优的使用体验，工程师、研究人员可以突破本地 HPC 基础设施限制，快速开始设计创新。
+HPC 高性能计算是山河 shanhe推出的一站式超算平台，提供弹性和可扩展的HPC计算资源，是传统HPC技术与云计算技术的融合升级。提供丰富多样的使用模式适配多种应用场景，统一的管理系统提供最优的使用体验，工程师、研究人员可以突破本地 HPC 基础设施限制，快速开始设计创新。
```
Comment (pethu1110) [inline content/compute/hpc/intro/intro.md:11]: 山河 shanhe?

File: content/compute/instance_group/news/product_news.md [hunk 9]
```diff
@@ -5,8 +5,8 @@ collapsible: false
 
 product:
     - time: 2019-10-24 17:50:47
-      title: 青云 QingCloud 推出安置策略组功能
-      content: 青云 QingCloud 针对用户的特定场景，一切从用户的角度出发，如提高资源的可用性、业务连续性，有效降低用户成本，基于用户感知的应用出发，开发了针对主机（instance）的安置策略组的功能。
+      title: 山河 shanhe 推出安置策略组功能
```
Comment (pethu1110) [inline content/compute/instance_group/news/product_news.md:8]: 山河 shanhe？
